docs/src/enterprise/configuration.md

```diff
@@ -17,6 +17,24 @@ The sections below provide additional configuration references that complement t
 
 This reference uses `.` to denote the nesting of values.
 
+### database
+
+Configuration section for lakeFS Enterprise database options.
+
+#### database.redis
```

Comment:
I'd perhaps add a couple of words of caution - writes need to be made durable either using AOF (`appendfsync always`), Using Redis Flex, MemoryDB, etc.